pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -734,10 +734,6 @@ default-package-overrides:
   - errors-ext ==0.4.2
   - ersatz ==0.4.8
   - esqueleto ==3.3.3.2
-  - essence-of-live-coding ==0.1.0.3
-  - essence-of-live-coding-gloss ==0.1.0.3
-  - essence-of-live-coding-pulse ==0.1.0.3
-  - essence-of-live-coding-quickcheck ==0.1.0.3
```

Comment:
@turion Did you mean to delete these?

These set the package versions for `essence-of-live-coding-*` to the versions from the Stackage LTS (and shouldn't be deleted).

Reply:
Oh, I misunderstood that then. Thanks for pointing out! I always have trouble finding my way around in this very long file.

I guess one could refactor it and put the list of broken packages in a separate file, so one always finds them.

Reply:
I reverted this.